workflow-job

```diff
@@ -598,7 +598,7 @@ flows["${sha1}"] = {
             sh "docker login -u ${dUser} -p ${dPassword}"
 
             sh "docker tag ${images.jamesCassandraRabbitMQ} linagora/james-cassandra-rabbitmq-swift:${tagName}"
-            sh "docker push linagora/james-project:${tagName}"
+            sh "docker push linagora/james-cassandra-rabbitmq-swift:${tagName}"
```

Comment:
it does not match https://github.com/linagora/james-project/blob/master/dockerfiles/packaging/guice/cassandra/Dockerfile#L3 . Do you plan to change it also?

Reply:
No let's keep `james-rabbitmq-project` as an image name. I did not understand that we had things already depending on it...